.rubocop.yml

```diff
@@ -78,6 +78,18 @@ RSpec/InstanceVariable:
 RSpec/NamedSubject:
   Enabled: false
 
+RSpec/LetSetup:
+  Exclude:
```

Comment:
I'm not sure why we don't just fix these, but no matter what, we should not override them here.  Use the #rubocop:disable syntax at point of violation instead please.

Reply:
my mistake, I think I copied these over from another app by accident